cmd/snapctl/snapctl.rst

```diff
@@ -0,0 +1,81 @@
+=========
+ snapctl
+=========
+
+-------------------------------------
+tool for accessing snap configuration
+-------------------------------------
+
+:Author: pawel.stolowski@canonical.com
+:Date:   2017-09-19
+:Copyright: Canonical Ltd.
+:Version: 1.0.0
```

Comment:
Let's use the snapd version that this was describing here

Reply:
Done.